D/Deno/build_tarballs.jl

```diff
@@ -27,6 +29,7 @@ install -m 755 "${target}/deno${exeext}" "${bindir}"
 # platforms are passed in on the command line
 platforms = [
     Platform("x86_64", "linux"; libc="glibc"),
+    Platform("aarch64", "linux"; libc="glibc"),
```

Comment:
I just copied the `libc` keyword argument from x86 linux, but do we also need it here?

Reply:
Yes